tests/energycal_test.py

```diff
@@ -276,3 +346,10 @@ def test_rm_calibration_error(uncal_spec):
     assert not uncal_spec.is_calibrated
     uncal_spec.rm_calibration()
     assert not uncal_spec.is_calibrated
+
+def test_callibration_not_inizialized_error(uncal_spec):
```

Comment:
Please check spelling :)

Reply:
Let me know if you find another one. Changed  a few docu strings in the tests.